src/app/builder/builder.component.html

```diff
@@ -25,6 +25,16 @@
       width="600"
       [src]="previewIframeSrc"></iframe>
   </div>
+  <div *ngIf="playPlaylist" id="preview-playlist-wrap">
```

Comment:
interesting, I think the root issue is that the preview iframe src just doesn't update when a specific episode is selected. Will handle now.